### ISSUE TITLE
Read/unread feed indicator

### ISSUE THREAD

**Issue by 92Julian** [adamsanderson/brook#7]
When an unread feed is selected, it is marked as read despite it has unread articles. After an update, the selected feed changes to unread and doesn't change to read when all articles are marked as read.

Feeds should remain unread if they have unread articles and change to read only when all articles are read.

**Comment by adamsanderson** [comment]
This is true, if some kind soul happens to come across this issue and wants to fix it, go for it!

That said, merely clicking on the feed again will clear it so it hasn't been enough of an issue to irritate me into fixing it all the years I've used it.
